s3aws.go:68 panic: runtime error: invalid memory address or nil pointer dereference [recovered]

After an upgrade to Complainer v1.4 i see in stderr:

```
I0711 14:14:21.902189  5962 exec.cpp:143] Version: 0.28.1
I0711 14:14:21.906971  5976 exec.cpp:217] Executor registered on slave e739267b-454f-4b63-82ba-c9918a4eae07-S1
WARNING: Your kernel does not support memory limit capabilities. Limitation discarded.
2016/07/11 14:14:22 Serving http on :8888
2016/07/11 14:15:04 Reporting ChronosTask:metrics-MetricsStoreBotcaptchaMetrics (ct:1468246502158:0:metrics-MetricsStoreBotcaptchaMetrics:) from 10.1.2.123
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x5ab6f6]

goroutine 1 [running]:
panic(0x9d8280, 0xc82000a0d0)
	/usr/lib/go/src/runtime/panic.go:481 +0x3e6
text/template.errRecover(0xc82016b4b0)
	/usr/lib/go/src/text/template/exec.go:133 +0xee
panic(0x9d8280, 0xc82000a0d0)
	/usr/lib/go/src/runtime/panic.go:443 +0x4e9
text/template.(*Template).Execute(0x0, 0x7fe5bc4df738, 0xc82012e460, 0x8d47e0, 0xc820329110, 0x0, 0x0)
	/usr/lib/go/src/text/template/exec.go:175 +0x226
github.com/cloudflare/complainer/uploader.(*s3AwsUploader).Upload(0xc82013a9f0, 0xc82042b280, 0x39, 0xc82042b2c0, 0x31, 0xc82018e706, 0xa, 0xc8203b15a0, 0xd, 0x0, ...)
	/go/src/github.com/cloudflare/complainer/uploader/s3aws.go:68 +0x25f
github.com/cloudflare/complainer/monitor.(*Monitor).processFailure(0xc820014d80, 0xc82042b280, 0x39, 0xc82042b2c0, 0x31, 0xc82018e706, 0xa, 0xc8203b15a0, 0xd, 0x0, ...)
	/go/src/github.com/cloudflare/complainer/monitor/monitor.go:160 +0x618
github.com/cloudflare/complainer/monitor.(*Monitor).Run(0xc820014d80, 0x0, 0x0)
	/go/src/github.com/cloudflare/complainer/monitor/monitor.go:101 +0x27b
main.main()
	/go/src/github.com/cloudflare/complainer/cmd/complainer/main.go:59 +0x93e
```

Marathon Config:
```json
{
  "id": "/complainer",
  "apps": [
    {
      "id": "/complainer/default",
      "cmd": "/go/bin/complainer",
      "cpus": 1,
      "mem": 2048.0,
      "disk": 50,
      "instances": 1,
      "container": {
        "type": "DOCKER",
        "docker": {
          "image": "cloudflare/complainer:1.4",
          "network": "BRIDGE",
          "portMappings": [
            {
              "containerPort": 8888,
              "hostPort": 0,
              "servicePort": 10888,
              "protocol": "tcp"
            }
          ]
        }
      },
      "env": {
          "COMPLAINER_MASTERS": "http://master1:5050,http://master2:5050,http://master3:5050",
          "COMPLAINER_REPORTERS": "slack",
          "COMPLAINER_UPLOADER": "s3aws",
          "COMPLAINER_LISTEN": ":8888",

          "S3_ACCESS_KEY": "ACCESS-KEY",
          "S3_SECRET_KEY": "SECRET-KEY",
          "S3_REGION": "eu-central-1",
          "S3_BUCKET": "BUCKET",

          "SLACK_HOOK_URL": "SLACK-HOOK-URL",
          "SLACK_USERNAME": "Mesos Cluster",
          "SLACK_CHANNEL": "#mesos",
          "SLACK_ICON_EMOJI": ":mesos:",
          "SLACK_FORMAT": "Task {{ .failure.Name }} ({{ .failure.ID }}) died with status {{ .failure.State }} [<{{ .stdoutURL }}|stdout>, <{{ .stderrURL }}|stderr>]{{if config \"mentions\"}} | {{ config \"mentions\" }}{{ .nl }}{{end}}"
      },
      "healthChecks": [
          {
              "protocol": "HTTP",
              "path": "/health",
              "gracePeriodSeconds": 10,
              "intervalSeconds": 60,
              "portIndex": 0,
              "timeoutSeconds": 10,
              "maxConsecutiveFailures": 3
          }
      ]
    }
  ]
}
```

--- a/uploader/s3aws.go
+++ b/uploader/s3aws.go
@@ -53,6 +53,11 @@ func newS3AwsUploader(accessKey, secretKey, region, bucket, prefix string, timeo
 		return nil, errors.New("s3 configuration is incomplete")
 	}
 
+	tmpl, err := template.New("").Parse(prefix)
+	if err != nil {
+		return nil, err
+	}
+
 	return &s3AwsUploader{
 		s3: s3.New(session.New(&aws.Config{
 			Region:      aws.String(region),
@@ -60,6 +65,7 @@ func newS3AwsUploader(accessKey, secretKey, region, bucket, prefix string, timeo
 		})),
 		bucket:  bucket,
 		timeout: timeout,
+		prefix:  tmpl,
 	}, nil
 }
 

